parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -39,22 +47,23 @@ fi
 
 if [ -f /var/run/reboot-required ]; then
     REBOOTREQUIRED=true
+    trace_info "RebootRequired" "reboot=true"
 else
     REBOOTREQUIRED=false
 fi
 
 {{- if not NeedsContainerd}}
-cleanUpContainerd
+time_metric "CleanupContainerd" cleanUpContainerd
```

Comment:
@palma21 FYI, AKS Engine CSE is now instrumented for App Insights telemetry. The `time_metric` func above is a wrapper around all CSE step executions, which is disabled by default, and is a no-op.

Just FYI.